### ISSUE TITLE
Facebook switching to WebSockets

### ISSUE THREAD

**Issue by gamelaster** [Schmavery/facebook-chat-api#719]
![image](https://user-images.githubusercontent.com/1665373/55264575-2829f380-5275-11e9-8497-77d499e06561.png)
but it's broken at the moment. Maybe MQTT implementation will be worth to do, since api chanes aaa lot

**Comment by ravkr** [comment]
yeah, but I've seen these websockets long ago (maybe 7-8 months), they might switch, but we will have to wait some time (long time...)

**Comment by 7osCraft** [comment]
Any news on this? I see that messenger still uses this WebSocket url.

**Comment by 7osCraft** [comment]
This should be fixed and implemented in #765

**Comment by ravkr** [comment]
@Schmavery you can close it now
